README.md

```diff
@@ -110,6 +110,18 @@ REMOTE_EXCLUDE=(ting styles advagg_*)
 LOCAL_PATH='sites/default/files'
 ```
 
+## SSL certification support
+
+This setup comes with self-signed wildcard certificates for *.local.itkdev.dk, but other certificates can be generated by using this openssl command on Mac, just change the two places where the domain is written.
+
+```sh
+openssl req -x509 -sha256 -nodes -days 3650 -newkey rsa:2048 -keyout docker.key -out docker.crt -subj "/CN=*.local.itkdev.dk" -reqexts SAN -extensions SAN -config <(cat /usr/local/etc/openssl/openssl.cnf <(printf '[SAN]\nsubjectAltName=DNS:*.local.itkdev.dk'))
+```
+
+To mac browser trust this certificate you need to open it with key-chain right click and select "Get info" and then open the "Trust" tab and select "Always trust".
```

Comment:
To make your browser trust this certificate you must add i locally. On a Mac open it with the keychain app. Then right [option] click on the certificate and select "Get info" and then open the "Trust" tab and select "Always trust".

Reply:
One note: Which certificate? The Readme should specify that is is `traefik/ssl/docker.crt` you need to add. And there's a command for it: `sudo security add-trusted-cert -d -r trustRoot -k "/Library/Keychains/System.keychain" myCA.pem
`